Sources/SwiftInspectorVisitors/NestableTypeVisitor.swift

```diff
@@ -159,14 +159,18 @@ public final class NestableTypeVisitor: SyntaxVisitor {
         genericRequirementVisitor.walk(genericWhereClause)
       }
 
+      let propertyVisitor = PropertySyntaxVisitor()
```

Comment:
we're currently in the 
```
else {
      // Recursive case. This is the first top-level declaration we've come across.
```
branch of this method. We also need to `properties += declarationVisitor.properties` in the `Base case` `if` branch of this method above. Search for `typealiases += declarationVisitor.typealiases` to find the spot.

We should also add a test that we find a property in a nested nestable type.